modules/javafx.controls/src/main/java/javafx/scene/control/TreeTableView.java

```diff
@@ -1809,6 +1810,11 @@ boolean isSortingInProgress() {
         return sortingInProgress;
     }
 
+    private boolean sortTreeOfSelectedItems = true;
+    boolean isSortTreeOfSelectedItems() {
+        return sortTreeOfSelectedItems;
```

Comment:
This method / state attribute isn't really needed. The value is never set to anything other than `true`. I recommend removing it.

Reply:
Corrected in the next commit..